src/synopsys-action/tools-parameter.ts

```diff
@@ -52,14 +52,21 @@ export class SynopsysToolsParameter {
           serverUrl: inputs.POLARIS_SERVER_URL,
           application: {name: inputs.POLARIS_APPLICATION_NAME},
           project: {name: inputs.POLARIS_PROJECT_NAME},
-          assessment: {types: assessmentTypeArray}
+          assessment: {types: assessmentTypeArray},
+          branch: {parent: {}}
         }
       }
     }
 
     if (inputs.POLARIS_TRIAGE) {
       polData.data.polaris.triage = inputs.POLARIS_TRIAGE
     }
+    if (inputs.POLARIS_BRANCH_NAME) {
+      polData.data.polaris.branch.name = inputs.POLARIS_BRANCH_NAME
```

Comment:
Are we not reading the value from github environment variable if not passed by user?

Reply:
bridge is reading from github env